include/deal.II/fe/fe_q.h

```diff
@@ -564,6 +564,36 @@ class FE_Q : public FE_Q_Base<TensorProductPolynomials<dim>, dim, spacedim>
    * constructor that specifies the polynomial degree only. For selecting
    * equidistant nodes at <tt>fe_degree > 2</tt>, construct
    * <tt>FE_Q<dim>(QIterated<1>(QTrapezoid<1>(),fe_degree))</tt>.
+   *
+   * This constructor results in an FE_Q object whose basis functions are still
+   * polynomials of degree equal to the number of elements of the @p points
+   * argument minus one. In other words, the *space* so constructed is the
+   * same as if you had called `FE_Q<dim>(points.size()-1)`. What *is*
```

Comment:
Are you sure ``*is*`` renders correctly in doxygen?

Reply:
Pretty sure -- doxygen does markdown these days. We've used in other places as well in recent times.